readthedocs/rtd_tests/fixtures/spec/v2/schema.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Schema for Read the Docs configuration file",
+  "type": "object",
+  "additionalProperties": false
+  "properties": {
+    "basics": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+          "version": {
+            "enum('2')"
```

Comment:
This doesn't look like a thing from json schema, please check how to declare this kind of validations, don't copy the ones from yamale, they aren't the same.